src/tests/specs/smoke/smokeTest.spec.ts

```diff
@@ -139,11 +139,11 @@ describe('Planner Smoke Tests:', () => {
     expect(await planner.quickPreview.hasComment('new comment')).toBeFalsy();
   });
 
-  xit('Create custom query', async() => {
+  it('Create custom query', async() => {
     await planner.sidePanel.clickRequirement();
     await planner.header.selectFilter('State','in progress');
     await planner.header.saveFilters('Query 1');
-    expect(await planner.sidePanel.hasCustomQuery('Query 1')).toBeTruthy();
+    expect(await planner.sidePanel.getMyFiltersList()).toContain('Query 1');
```

Comment:
hasCustomQuery('Query 1')).toBeTruthy();
and 
getMyFiltersList()).toContain('Query 1');
test for the same thing.
Why should we use getMyFiltersList()).toContain('Query 1') ?

Reply:
@nimishamukherjee The problem with using `hasCustomQuery()` methods is that they're difficult to debug. When you use the `hasCustomQuery()` method and the tests fails, you see 
`Expected false to be true`. This output is useless if you wish to know why the tests failed.
Instead, when we use the `getMyFiltersList()` we have output like `Expected ['a', 'b', 'c'] to contain 'd'` and you know why the tests failed.

We will be refactoring all the `has...()` methods to `get...()` method.